src/main/kotlin/no/nav/familie/ef/sak/behandling/BarnController.kt

```diff
@@ -19,11 +19,19 @@ import java.util.UUID
 @Validated
 class BarnController(val nyeBarnService: NyeBarnService) {
 
+    @Deprecated("nye-eller-tidligere-fodte-barn")
     @PostMapping("nye-barn")
     fun finnNyeBarnSidenGjeldendeBehandlingForPerson(@RequestBody personIdent: PersonIdent): Ressurs<List<String>> {
         return Ressurs.success(nyeBarnService.finnNyeBarnSidenGjeldendeBehandlingForPersonIdent(personIdent))
     }
 
+    @PostMapping("nye-eller-tidligere-fodte-barn")
+    // for å unngå att vi oppretter oppgaver for nye barn så sjekkes roles
+    @ProtectedWithClaims(issuer = "azuread", claimMap = ["roles=access_as_application"])
```

Comment:
Denne skjønte jeg ikke - hva skjer her?

Reply:
Mener du;
`for å unngå at utviklere oppretter oppgaver...` ?

Reply:
Det sjekker att det er en applikasjon som kaller ef-sak, og ikke on_behalf (saksbehandler via ef-sak-frontend)
Denne burde kanskje vært under ekstern endepunkt

Reply:
Endrer til
"// denne skal kalles på fra ef-personhendelse for å opprette oppgaver for nye eller for tidligt fødte barn"